### ISSUE TITLE
Playlist isn't generated for already downloaded podcast

### ISSUE THREAD

**Issue by ElonSatoshi** [mlj/castget#25]
How to reproduce issue:

Put in ~/.castgetrc: (replace home directory with your own home directory)

`[hpmor]
url=https://feeds.feedburner.com/HarryPotterAndTheMethodsOfRationalityThePodcast
spool=/home/elonsatoshi/Audio/MethodsOfRationality/`

Run castget.

When podcast is finished downloading, add playlist: 

`[hpmor]
url=https://feeds.feedburner.com/HarryPotterAndTheMethodsOfRationalityThePodcast
spool=/home/elonsatoshi/Audio/MethodsOfRationality/
playlist=/home/elonsatoshi/Audio/MethodsOfRationality.m3u`

Run `ls ~/Audio/MethodsOfRationality.m3u`

Expected result:

`/home/elonsatoshi/Audio/MethodsOfRationality.m3u`

Actual result:

`ls: cannot access '/home/elonsatoshi/Audio/MethodsOfRationality.m3u': No such file or directory`

**Comment by mlj** [comment]
Not sure I understand: Do you mean that if you run `castget` between adding `playlist` to `.castgetrc` and running `ls`, you expect to find a generated playlist?

**Comment by ElonSatoshi** [comment]
Yes. I expect `castget` to generate a playlist that includes any previously downloaded files.
